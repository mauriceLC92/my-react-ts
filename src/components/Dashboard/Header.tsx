import * as React from "react";

const logo = require('../../images/logo.svg');

interface State {

}

interface Props {

}


export class Header extends React.Component<Props, State>{
    render(){
        return(
            <div className="main-header">
            <header className="App-header">
                <img src={logo} className="App-logo" alt="logo" />
                <h1 className="App-title">Kurtosys Horse Race Challenge</h1>
            </header>
            <div>
            <i className="wheelchair icon"></i>...........................................KURTOSYS 1000M Pixel Dash.....................................<i className="wheelchair icon"></i>
            </div>
            </div>
        );
    }
}