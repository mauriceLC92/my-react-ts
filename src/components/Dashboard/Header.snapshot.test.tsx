import * as React from 'react';
const renderer =require('react-test-renderer');
import {Header} from './Header';

describe('Header component renders the header correctly', () => {
  it('renders correctly', () => {
    
    const rendered = renderer.create(
      <Header />
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});