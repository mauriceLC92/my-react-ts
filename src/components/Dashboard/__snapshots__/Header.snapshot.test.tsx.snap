// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders the header correctly renders correctly 1`] = `
<div
  className="main-header"
>
  <header
    className="App-header"
  >
    <h1
      className="App-title"
    >
      Kurtosys Horse Race Challenge
    </h1>
  </header>
  <div>
    <i
      className="wheelchair icon"
    >
      .
    </i>
    ..,KURTOSYS 1000M Pixel Dash..,
    <i
      className="wheelchair icon"
    >
      .
    </i>
  </div>
</div>
`;
